src/pynws/nws.py

```diff
@@ -62,6 +63,11 @@ def __init__(
         self.county_zone: Optional[str] = None
         self.fire_weather_zone: Optional[str] = None
 
+        if forecast_units in ["us", "si"]:
```

Comment:
I'd prefer to see a tuple instead of a list here: `if forecast_units in ("us", "si"):`